maven-model-builder/src/main/java/org/apache/maven/model/building/ModelBuildingRequest.java

```diff
@@ -57,10 +57,15 @@ public interface ModelBuildingRequest {
      */
     int VALIDATION_LEVEL_MAVEN_3_1 = 31;
 
+    /**
+     * Denotes validation as performed by Maven 3.1. This validation level is meant for new projects.
```

Comment:
```suggestion
     * Denotes validation as performed by Maven 4.0. This validation level is meant for new projects.
```

Reply:
Also, for `VALIDATION_LEVEL_MAVEN_3_1` I think the Javadoc should now read "This validation level is meant for existing projects."